android_splash_screen/pubspec.yaml

```diff
@@ -0,0 +1,77 @@
+name: splash_screen_sample
+description: A new Flutter project.
```

Comment:
Add a brief, custom description here.

Also, you can remove any auto-generated comments from this file, as well as anything you're not using, like cupertino_icons.